src/main/scala/mesosphere/marathon/raml/GroupConversion.scala

```diff
@@ -128,7 +128,7 @@ object UpdateGroupStructureOp {
       groupsById = effectiveGroups,
       dependencies = effectiveDependencies,
       version = timestamp,
-      enforceRole = groupUpdate.enforceRole.get
+      enforceRole = groupUpdate.enforceRole
```

Comment:
On this not: While looking into a COPS issue, I noticed that there's another creation of Group going on in line 72-77 and we're not setting enforceRole there. Is that intentional or did we just miss it?

Reply:
Yessir, this fix is in a separate PR. See https://jira.d2iq.com/browse/MARATHON-8770